docs/source/api/plugin/landing-pages.md

```diff
@@ -214,6 +214,21 @@ If provided, these variables should apply to the initial query you provide in `d
 
 An object containing initial HTTP header values to populate in the Explorer on load.
 
+</td>
+</tr>
+<tr>
+<td>
+
+###### `includeCookies`
+
+`boolean`
+</td>
+<td>
+
+A boolean used to set whether cookies are included in the Studio Explorer's editor on load.
```

Comment:
I think this could a bit more explicit, like, "A boolean used to set whether Studio Explorer should include cookies in its GraphQL requests to your server." ?